docker/src/main/java/brooklyn/entity/container/docker/DockerContainerImpl.java

```diff
@@ -28,6 +28,7 @@
 
 import javax.annotation.Nullable;
 
+import com.google.common.collect.*;
```

Comment:
hmm, must have made this commit before I changed the IDE settings. Will go through all classes and check when we are ready to merge